src/turbomind/kernels/attention/attention_universal.h

```diff
@@ -605,8 +580,8 @@ struct AttentionUniversal {
         Impl::ForeachML(frag_M, frag_L, [&](int hi, int qi, int ri, float M, float L) {
             const int index = get_index(hi, qi);
             if (qi_begin + qi < qi_end && ri == 0 && check_h(hi)) {
-                params.partial_M[index] = M;
-                params.partial_L[index] = L;
+                params.partial_ML[index * 2]     = M;
```

Comment:
make the `partial_ML` a pointer to `float2` so that load / store can be vectorized.